docs_app/src/app/custom-elements/migration-timeline/components/migration-timeline/breaking-change-description-table.component.ts

```diff
@@ -0,0 +1,61 @@
+import {Component, Input, Output} from '@angular/core';
+import {Subject} from 'rxjs';
+import {ClientBreakingChange} from '../../data-access/migration-item';
+import {State} from '../../utils/state.service';
+
+interface VMBreakingChangeDescriptionTable {
+  breakingChange: ClientBreakingChange;
+}
+
+@Component({
+  selector: `breaking-change-description-table`,
+  template: `
+    <table *ngIf="vm$ | async as vm">
```

Comment:
please use a seperate file for complex html files

Reply:
Made some refactoring of  migration-timeline component.
Introdued several new smaller components.
https://github.com/ReactiveX/rxjs/pull/5128/commits/464f565097e73598d9cf3bec0caaceb980f963f3